chapters/sparse.rst

```diff
@@ -0,0 +1,129 @@
+Use Sparse Embedding in Jina
+==============================
+
+.. meta::
+   :description: Sparse Embedding in Jina
+   :keywords: Jina, sparse, coo, csr
+
+.. contents:: Table of Contents
+    :depth: 2
+
+Motivation
+------------
+
+A sparse matrix is a special case of a matrix in which the number of zero elements is much higher than the number of non-zero elements.
+The space used for representing data and the time for scanning the matrix can be reduced significantly using a sparse representation.
+In this guideline, we'll introduce how to use sparse matrix in Jina.
+
+Before you start
+----------------
+
+Before you begin, make sure you meet these prerequisites:
+
+-  Make sure you have basic understanding on Jina.
+-  Make sure you have basic understanding on sparse matrices.
+-  We assume you have some experience with `sparse module in scipy <https://docs.scipy.org/doc/scipy/reference/sparse.html>`_, or you have used `Tnesorflow Sparse Tensor <https://www.tensorflow.org/api_docs/python/tf/sparse/SparseTensor>`_ / `PyTorch Sparse COO Tensor <https://pytorch.org/docs/stable/sparse.html#sparse-coo-tensors>`_.
+
+.. Note::
+    This feature is not yet production-ready.
```

Comment:
I would say the `sparse indexers` in the `hub` do not support ACID features

Reply:
@JoanFM I'll add to limitation section